src/lib/core/public-api.ts

```diff
@@ -21,6 +21,7 @@ export {
   MockMatchMediaProvider as ɵMockMatchMediaProvider,
 } from './match-media/index';
 export * from './media-observer/index';
+export * from './media-trigger/index';
```

Comment:
If this is a private re-export, it needs to be the following:

```ts
export {MediaTrigger as ɵMediaTrigger} from './media-trigger/index';
```

Reply:
MediaTrigger is public. For Designer usages.